app/controllers/projects_controller.rb

```diff
@@ -179,7 +178,7 @@ def remove_user_from_project
   private
 
   def load_pagy
-    @pagy, @projects = pagy(@projects.order(created_at: :desc), link_extra: "data-remote='true'", items: Company::PAGE_SIZE)
+    @pagy, @projects = pagy(@projects, link_extra: "data-remote='true'", items: Company::PAGE_SIZE)
```

Comment:
Here this might cause inconsistency as pagination is done in multiple actions. So this should either be done as a default_scope or removed